### ISSUE TITLE
PAT projects created on Windows don't run successfully on Mac

### ISSUE THREAD

**Issue by asparke2** [NREL/OpenStudio#2540]
Note: PAT projects created on Mac **DO** run successfully on Windows.

I suspect that this is a bug in how path strings are being set or concatenated.

Steps to reproduce:

1. Create a PAT project on Windows
2. Zip it and send to Mac
3. Unzip the file on Mac
4. Open the project in PAT
5. Run

In the following example, `aparker` is Windows and `dgoldwas` is Mac.  Ignore the diff order.

Diffing the pat.json (after opening the project on Windows), we can see that the `projectDir` was correctly changed.  However, instead of replacing the `measure_dir` and `directory` correctly, PAT added the Mac path to the beginning of the string.

```diff
   "projectName": "Test One Failing Run 1",
-  "projectDir": "/Users/dgoldwas/Documents/Test One Failing Run 1",
+  "projectDir": "C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1",
   "seed": "medium_office_2007.osm",
   "weatherFile": "USA_TX_Houston-Bush.Intercontinental.AP.722430_TMY3.epw",
   "analysis_type": "Manual",
@@ -45,9 +45,9 @@
   "openstudioMD5": "",
   "measures": [
     {
-      "measure_dir": "/Users/dgoldwas/Documents/Test One Failing Run 1/measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
+      "measure_dir": "C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
       "name": "reduce_night_time_electric_equipment_loads",
-      "directory": "/Users/dgoldwas/Documents/Test One Failing Run 1/measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
+      "directory": "C:/GitRepos/OpenStudio-measures/NREL working measures/ReduceNightTimeElectricEquipmentLoads",
       "uid": "7bde0d05-3714-43e2-91b4-aa1e12e714bf",
       "uuid": "{7bde0d05-3714-43e2-91b4-aa1e12e714bf}",
       "version_id": "3a98256e-545d-4f22-abf3-121182e882c8”,
```

This leads to incorrect paths in the `analysis.json` that gets sent to the server.  When the server looks for the measures in these paths, it can't find them and the runs fail.

```diff
           "display_name": "Reduce Night Time Electric Equipment Loads",
           "measure_type": "RubyMeasure",
           "measure_definition_class_name": "ReduceNightTimeElectricEquipmentLoads",
-          "measure_definition_directory": "./measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
-          "measure_definition_directory_local": "/Users/dgoldwas/Documents/Test One Failing Run 1/measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
+          "measure_definition_directory": "./measures/ReduceNightTimeElectricEquipmentLoads",
+          "measure_definition_directory_local": "C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads",
           "measure_definition_display_name": "Reduce Night Time Electric Equipment Loads",
           "measure_definition_name": "reduce_night_time_electric_equipment_loads",
           "measure_definition_name_xml": null,
@@ -214,8 +214,8 @@
           "display_name": "OpenStudio Results",
           "measure_type": "ReportingMeasure",
           "measure_definition_class_name": "OpenStudioResults",
-          "measure_definition_directory": "./measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\openstudio_results",
-          "measure_definition_directory_local": "/Users/dgoldwas/Documents/Test One Failing Run 1/measures/C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\openstudio_results",
+          "measure_definition_directory": "./measures/openstudio_results",
+          "measure_definition_directory_local": "C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\openstudio_results",
           "measure_definition_display_name": "OpenStudio Results",
           "measure_definition_name": "openstudio_results",
           "measure_definition_name_xml": null,
```

During the run, the data_points/12341233.json indicates that the measure can't be found.
```
[13:58:56.188807 ERROR] Found error in state 'initialization' with message ["Cannot find measure C:\\Users\\aparker\\Desktop\\Testing\\PAT 2.0.5_3 Report Fix\\Test One Failing Run 1\\measures\\ReduceNightTimeElectricEquipmentLoads:/Applications/OpenStudio-2.0.5/ParametricAnalysisTool.app/Contents/Resources/OpenStudio-server/gems/gems/openstudio-workflow-1.2.0/lib/openstudio/workflow/util/measure.rb:108:in `block in validate_measures'
```

**Comment by asparke2** [comment]
@kflemin This is not the same thing as the `ERROR EOFError: bad content body` error we were discussing.  Still don't know what caused that one.

**Comment by kflemin** [comment]
just committed a fix

**Comment by asparke2** [comment]
Confirmed fixed w/ 2.1.1 by @DavidGoldwasser on Mac
